frontend/src/app/components/admin-profile/admin-profile.component.html

```diff
@@ -53,14 +53,14 @@ <h1>Admin Profile</h1>
     <div class="flex-container">
         <div class="left">
 
-            <mat-form-field class="inputField-med">
+            <mat-form-field class="inputField-med" disabled>
```

Comment:
I'm wondering about the addition of the disabled keyword. In admin-profile.component.ts, when we define the FormGroup, passing in disabled:true disabled the input field in an earlier version of this component

Reply:
This was probably an accident, sorry.

Reply:
No worries, then go ahead and delete this keyword from the mat form fields and cross check that the respective FormGroup field has a disabled:true. The password won’t have this because the user might want to change their password